README.md

```diff
@@ -1,2 +1,12 @@
 # conformance-status-page
 Web application displaying data availability and profile conformity of OpenActive endpoints
+
+Using `node` version `13.9`. Project uses standard `express-generator` conventions.
+
+# Install
+
+`$ npm install`
+
+# Run the app (developer)
+
+`$ npm start-dev `
```

Comment:
Looks like "npm start" is prod command, can add that? (But maybe in separate P.R. is fine?)